rdrf/rdrf/settings.py

```diff
@@ -53,11 +53,17 @@
 LANGUAGE_CODE = env.get("language_code", 'en')
 USE_I18N = env.get("use_i18n", True)
 
-LANGUAGES = (
-    ('ar', 'Arabic'),
-    ('de', 'German'),
-    ('en', 'English'),
-)
+# This must be a superset of LANGUAGES
```

Comment:
Why is this needed? More doc?
It says that it has to be a superset of `LANGUAGES`, but `LANGUAGES` is read from the environment and the "superset relationship" isn't verified.
Is that important?